### ISSUE TITLE
AppImage  puking on File/open

### ISSUE THREAD

**Issue by BusinessKid** [Ultimaker/Cura#15706]
### Application Version

5.2.1 & 5.3.1

### Operating System

Linux Slacikware-15.0

### Printer

Anycubic I3 Mega X

### Reproduction steps

1. Start UltiMaker Cura
2. Click on any File/open icon or menu item.

### Actual results

After investigation, the process Ultimaker-Cura dies noiselessly. That apparently collapses the AppRun script. I get this error in a terminal

Some other Slackware-15.0 installs work ok. Slackware-15.0 uses /lib64 & /usr/lib64 for 64bit libs. It offers KDE or XFCE, and Xorg or Wayland. It does not install systemd. The packages are the o/p of 'make install' so you get all libs, includes, docs & share files. 

### Expected results

I should have got a file/open dialog.

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Additional information & file uploads

bash-5.1$ /opt/UltiMaker-Cura-5.3.1-linux.AppImage
PyInstaller/loader/pyimod02_importers.py:352: DeprecationWarning: the imp module is deprecated in favour of importlib and slated for removal in Python 3.12; see the module's documentation for alternative uses
Gtk-Message: 17:04:55.154: Failed to load module "colorreload-gtk-module"
Gtk-Message: 17:04:55.154: Failed to load module "window-decorations-gtk-module"
Fontconfig warning: "/etc/fonts/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
Cyclic dependency detected between "file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/Actions.qml" and "file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/Actions.qml"
QQmlEngine::setContextForObject(): Object already has a QQmlContext
file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/PrintSetupSelector/Recommended/RecommendedPrintSetup.qml:96:17: QML QQuickItem: Binding loop detected for property "height"
file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/PrintSetupSelector/Recommended/RecommendedPrintSetup.qml:96:17: QML QQuickItem: Binding loop detected for property "height"

(UltiMaker-Cura:10162): Gtk-WARNING **: 17:14:09.000: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Tango/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Tango/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
####End of Startup terminal output. Next action is click on file/open####
/tmp/.mount_UltiMapAP0aG/AppRun: line 25: 9815 Aborted                 $scriptdir/UltiMaker-Cura "$@"

Process 10162 is UltiMaker-Cura. I ran a 'ps -e' before hitting file/open elsewhere to identify it. Here's the latest attempt from the log

**Comment by BusinessKid** [comment]
That log file segment:
2023-06-09 17:09:33,391 - DEBUG - [MainThread] UM.TaskManagement.HttpRequestManager._onRequestError [369]: request[8cdfd3da][get][PyQt6.QtCore.QUrl('https://api.ultimaker.com/cura-drive/v1/backups')][timeout=None][no-data] got an QNetworkReplyError Error transferring https://api.ultimaker.com/cura-drive/v1/backups - server replied: . The server returned: b'{"errors": [{"id": "d10c84a3-d0f5-4bb5-9a78-28a89cdc65fc", "code": "noBearerToken", "http_status": "401", "title": "No bearer token has been given to authenticate the request.", "meta": {"error_type": "AuthError", "url": "https://api.ultimaker.com/backups"}}]}'
2023-06-09 17:09:33,391 - WARNING - [MainThread] CuraDrive.src.DriveApiService.callback [41]: Could not get backups: NetworkError.AuthenticationRequiredError
2023-06-09 17:09:50,288 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [444]: file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/MainWindow/ApplicationMenu.qml:53:9: QML ViewMenu: Binding loop detected for property "width"
2023-06-09 17:09:50,288 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [444]: file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/MainWindow/ApplicationMenu.qml:53:9: QML ViewMenu: Binding loop detected for property "width"
2023-06-09 17:10:32,187 - DEBUG - [MainThread] cura.AutoSave._onTimeout [64]: Autosaving preferences, instances and profiles took 0.006530046463012695 seconds
2023-06-09 17:10:51,936 - DEBUG - [MainThread] UM.Controller.setActiveStage [180]: Setting active stage to PreviewStage
2023-06-09 17:10:51,948 - DEBUG - [MainThread] UM.Controller.setActiveView [108]: Setting active view to SimulationView
2023-06-09 17:10:51,950 - DEBUG - [MainThread] UM.Mesh.MeshData.calculateNormalsFromVertices [561]: Calculating normals took 0.0002961158752441406 seconds
2023-06-09 17:10:51,950 - DEBUG - [MainThread] STLReader.STLReader._read [68]: Loaded a mesh with 12612 vertices
2023-06-09 17:10:51,951 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/plugins/SimulationView/simulationview_composite.shader]...
2023-06-09 17:10:52,019 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/uranium/resources/shaders/color.shader]...
2023-06-09 17:10:52,021 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/plugins/SimulationView/layers3d.shader]...
2023-06-09 17:10:52,049 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/plugins/SimulationView/layers3d_shadow.shader]...
2023-06-09 17:10:52,072 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/uranium/resources/shaders/toolhandle.shader]...
2023-06-09 17:10:52,074 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/uranium/resources/shaders/color.shader]...
2023-06-09 17:10:52,075 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/resources/shaders/striped.shader]...
2023-06-09 17:11:02,184 - DEBUG - [MainThread] cura.AutoSave._onTimeout [64]: Autosaving preferences, instances and profiles took 0.006354808807373047 seconds
2023-06-09 17:11:08,599 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [444]: file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/MainWindow/ApplicationMenu.qml:51:9: QML FileMenu: Binding loop detected for property "width"
2023-06-09 17:11:19,166 - DEBUG - [MainThread] UM.Controller.setActiveView [108]: Setting active view to XRayView
2023-06-09 17:11:19,167 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/resources/shaders/xray_composite.shader]...
2023-06-09 17:11:19,174 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/resources/shaders/xray.shader]...
2023-06-09 17:11:19,175 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [/tmp/.mount_UltiMapAP0aG/share/cura/resources/shaders/xray.shader]...
2023-06-09 17:11:24,758 - DEBUG - [MainThread] UM.Controller.setActiveView [108]: Setting active view to SimulationView
2023-06-09 17:11:46,184 - DEBUG - [MainThread] cura.AutoSave._onTimeout [64]: Autosaving preferences, instances and profiles took 0.006392478942871094 seconds
2023-06-09 17:11:47,511 - DEBUG - [MainThread] UM.Controller.setActiveView [108]: Setting active view to XRayView
2023-06-09 17:14:03,538 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [444]: file:///tmp/.mount_UltiMapAP0aG/share/cura/resources/qml/MainWindow/ApplicationMenu.qml:51:9: QML FileMenu: Binding loop detected for property "width"

**Comment by BusinessKid** [comment]
Boy, this release looks like a real POS with all these bugs coming thick & fast. This was solved for me. Apparently installing some xdg-desktop-portal app is needed with the XFCE Window manager. I looked in the app. It's adds freedesktop.portal xml files to dbus which makes no sense to me at all.  This thread refers:[https://www.linuxquestions.org/questions/slackware-14/ultimaker-cura-appimages-puking-on-slackware-15-0-a-4175725837/](url) Post 22/23 are the interesting ones. It was being a bit promiscuous between GTK & QT. I's definitely "WORKS.FOR.ME" and not solved.
